client/my-sites/plans/index.js

```diff
@@ -11,7 +11,23 @@ import controller from 'my-sites/controller';
 import paths from './paths';
 import plansController from './controller';
 import { retarget } from 'analytics/ad-tracking';
-import googleAnalyticsLandingPage from './features/google-analytics';
+import googleAnalyticsLandingPage from './plans-features/google-analytics';
+
+function newRouteWithSiteSelectionFallback( route, routeController ) {
```

Comment:
I really don't like these helpers that gradually make telling what is matching a given route a pain.